modules/llama_cpp_plugin/notebooks/llama_cpp_plugin_with_qwen.ipynb

```diff
@@ -0,0 +1,276 @@
+{
+ "cells": [
```

Comment:
Since the path to the notebook already contains "llama_cpp_plugin" name you can shorten its name to `qwen.ipynb`

Reply:
Done